content/zh-cn/privacy.md

```diff
@@ -2,63 +2,99 @@
 
-此外，我们使用 Google Analytics 来估计网站上的流量和热门网页。作为该服务的一部分，我们会在我们的网站上放置 Google Analytics 的 Cookies。这些 Cookies 不包含个人信息，但可以在我们的网站上随时间唯一识别您的浏览器软件。我们尊重 [Do Not Track](http://donottrack.us/) 请求标头，为所有访客严格限制我们的分析服务可以收集和分享的信息。
+## 我们不会出售您的数据或信息
+
+我们不会出售您的数据或信息. 这包括依赖方、 订阅者和访客的数据和信息。
 
 ## 执法请求与情有可原的情况
 
-在我们拥有您的个人身份信息的期间内，我们可能会在有限的情况下向第三方披露这些信息。这些情况包括：当我们征得您的同意或我们真诚地相信法律要求我们这样做时（例如根据传票或其他司法或行政命令）。当我们真诚地相信有必要防止生命损失、人身伤害、财产损失或重大财务损失时，我们可能也会披露帐户恢复信息。
+在我们拥有您的个人身份信息的期间内，我们可能会在有限的情况下向第三方披露这些信息。 这些情况包括：当我们征得您的同意或我们真诚地相信法律要求我们这样做时（例如根据传票或其他司法或行政命令）。 当我们真诚地相信有必要防止生命损失、人身伤害、财产损失或重大财务损失时，我们可能也会披露帐户恢复信息。
 
-如果法律要求我们披露您提交的信息，我们将尝试事先通知您这一情况（除非我们被禁止这样做，或者这样做是徒劳的），以给您反对披露信息的机会。我们将尝试通过任何合理可行的方式提供此通知。如果您不质疑披露请求，我们将会按法律要求提供您的信息。
+如果法律要求我们披露您提交的信息，我们将尝试事先通知您这一情况（除非我们被禁止这样做，或者这样做是徒劳的），以给您反对披露信息的机会。 我们将尝试通过任何合理可行的方式提供此通知。 如果您不对披露请求提出异议，我们将会按法律要求提供您的信息。
 
-此外，我们保留自行决定独立反对某些我们认为不合适的（访问我们产品和技术的用户的信息的）请求的权利。
+此外，我们保留自行决定独立反对某些我们认为不适当的请求（访问我们的产品和技术用户的信息）的权利。
 
 ## 欧洲经济区的信任方、用户和访客在 GDPR 下享有哪些权利？我该如何行使这些权利？
 
-我们按照本政策的规定处理个人数据。我们发送电子邮件前需要您的同意。当我们收集IP地址时，我们根据能够证明服务按预期执行的合同必要性处理该数据。请注意，我们可能无法删除包括IP地址在内的信息，因为在确定证书的可信度时，其他人必须依赖这些信息。在某些情况下，我们可能会根据法定义务或是保护您或其他人的重要利益而处理个人数据。
+我们按照本政策的规定处理个人数据。 资料处理的目的和法律依据如下：
+
+**目的：** 提供证书状态 (OCSP) 信息
+
+**法律依据：** 合法利益
+
+**额外信息：** 我们收集并处理来自依赖方的信息，以便可靠地提供证书状态信息。
+
+**目的：** 提供证书签发和管理服务
+
+**法律依据：** 合规合法利益
+
+**额外信息：** 我们收集并处理订阅者提供的信息，以便提供可靠和安全的证书签发和管理服务。 并向公众表明我们的服务正在按预期进行。
+
+**目的：** 向访客提供信息
+
+**法律依据：** 许可，合法利益
+
+**额外信息：** 我们收集和处理来自访客的信息，以便以可靠和有效的方式通过网络和电子邮件提供信息。
+
+**目的：** 处理捐赠和赞助调查问题
+
+**法律依据：** 合法利益
+
+**额外信息：** 我们收集和处理信息，以处理和支持捐赠。
+
+**目的：** 法律义务和情有可原的情况
+
+**法律依据：** 法定义务，合法利益
+
+**额外信息：** 我们可能会收集和处理信息以履行法律义务，或当我们真诚地相信有必要防止生命损失、人身伤害、财产损失或重大经济损失。
+
+请注意，我们可能无法删除包括IP地址在内的信息，因为在确定证书的可信度时，其他人必须依赖这些信息。 在某些情况下，我们可能会根据法定义务或是保护您或其他人的重要利益而处理个人数据。
+
+您的个人数据可能会被我们和我们的服务提供商收集或转移到存储或处理数据的司法管辖区（包括美国）。 这些法域可能无法提供与贵国法域（包括欧洲经济区）相同程度的数据保护。 我们已采取步骤确保我们的服务提供者为欧洲经济区居民的个人资料提供充分的保护。 包括使用欧洲联盟委员会核准的标准合同条款订立数据处理协定，或使用欧洲联盟委员会核准的其他保障措施。 您有权通过以下联系信息向我们发送电子邮件，以获取您的个人信息在欧盟以外传输的机制的详细信息。
 
-位于欧洲经济区（EEA）的个人对其个人信息拥有一定的权利，这包括访问、更正或删除我们通过您使用本网站处理的个人数据的权利。如果您是 EEA 的信任方、用户或访客，您可以：
+位于欧洲经济区（EEA）的个人对其个人信息拥有一定的权利，这包括访问、更正或删除我们通过您使用本网站处理的个人数据的权利。 如果您是位于 EEA 的信任方、用户或访客，您可以：
 
-* 通过发送电子邮件至 security@letsencrypt.org 索取个人数据报告。此报告将包含我们拥有的有关您的个人数据，这些数据以结构化、常用且可移植的格式提供给您。请注意，在我们披露任何信息之前，我们可能会要求您提供其他信息以验证您的身份。
+- 通过发送电子邮件至 security@letsencrypt.org 索取个人数据报告。 本报告将包括我们所掌握的关于您的个人数据，这些数据以结构化、常用和便捷的格式提供给您。 请注意，在我们披露任何信息之前，我们可能会要求您提供其他信息以验证您的身份。
 
-* 通过 security@letsencrypt.org 与我们联系，要求更正或删除您的信息。
+- 通过发送电子邮件至 security@letsencrypt.org 联系我们，要求更正或删除您的信息。
 
-* 反对我们处理您的信息。您可以要求我们停止使用您的信息，包括我们使用您的信息向您发送服务电子邮件。您可以随时通过点击 Let's Encrypt 电子邮件中的“取消订阅”链接，撤销对于接收服务电子邮件的许可。
+- 反对我们处理您的信息。 您可以要求我们停止使用您的信息，包括我们使用您的信息向您发送服务相关电子邮件。 您可以随时通过点击 Let's Encrypt 电子邮件中的“取消订阅”链接，撤销对于接收服务电子邮件的许可。
 
-* 向监管机构投诉。如果您位于欧洲经济区，并认为我们没有遵守数据保护法，您有权向当地监管机构提出投诉。
+- 向监管机构投诉。 如果您位于欧洲经济区，并认为我们没有遵守数据保护法，您有权向当地监管机构提出投诉。
 
-要获取更多信息或报告隐私问题，请联系：[security@letsencrypt.org](mailto:security@letsencrypt.org)
+了解更多信息或报告隐私问题，请联系：security@letsencrypt.org 。
```

Comment:
We should keep `[security@letsencrypt.org](mailto:security@letsencrypt.org)`, without it the link in broken:
![image](https://user-images.githubusercontent.com/1955774/198879572-16622da9-4d68-4654-be12-83a1849ae3cd.png)